account_journal_report/tests/test_account_journal_report.py

```diff
@@ -0,0 +1,88 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Eficent Business and IT Consulting Services S.L.
+#   (http://www.eficent.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from openerp.tests.common import TransactionCase
+
+
+class TestAccountJournalReport(TransactionCase):
```

Comment:
Use SavepointCase instead, that is more efficient. You need to switch setup method, but only that. You can see an example here: https://github.com/OCA/l10n-spain/blob/8.0/l10n_es_account_financial_report/tests/test_journal_entries_report.py